### PR TITLE
EC-218: Quick order, vouchers, payment revision

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://issues.ibexa.co/browse/EC-218
| Versions      | 3.1, 3.2, master

### CODE REVIEW

File: docs/guide/quick_order/quick_order_configuration.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+siso_search.default.search.auto_suggest_fields:
+    - ses_product_ses_sku_value_s
+    - ses_product_ses_name_value_s
+    - ses_variant_list_s
+    - ses_variant_codes_ms
+    - ses_variant_sku_ms
+    - ses_variant_desc_ms
+```
+
+The `ses_variant_*` fields are additional fields indexed in the Solr.
```
Comment (dabrt) [inline docs/guide/quick_order/quick_order_configuration.md:70]:
```suggestion
The `ses_variant_*` fields are additional fields that are indexed in Solr.
```

File: docs/guide/vouchers/voucher_templates.md [hunk 2]
```diff
@@ -1,14 +1,6 @@
...
-## Template list
-
-The block with vouchers is included in `Silversolutions/Bundle/EshopBundle/Resources/views/Basket/show.html.twig`:
-
-``` html+twig
-{% block vouchers %}
-...
-{% endblock %}
-```
+Vouchers are rendered in the `vouchers` Twig block included in `Silversolutions/Bundle/EshopBundle/Resources/views/Basket/show.html.twig`.
```
Comment (dabrt) [inline docs/guide/vouchers/voucher_templates.md:3]:
```suggestion
Vouchers are rendered in the `vouchers` Twig block that is included in `Silversolutions/Bundle/EshopBundle/Resources/views/Basket/show.html.twig`.
```

File: docs/guide/vouchers/vouchers.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+# Vouchers
+
+[[= product_name_com =]] supports vouchers that are managed in ERP. The customer can enter a voucher number in the basket.
+Then the voucher is sent to the ERP and, if it is valid, the customer gets a discount.
+
+Voucher data is sent to ERP in `PriceRequest` and `CreateOrderRequest`.
```
Comment (dabrt) [inline docs/guide/vouchers/vouchers.md:6]:
```suggestion
Voucher data is sent to the ERP in `PriceRequest` and `CreateOrderRequest`.
```

File: docs/guide/vouchers/vouchers.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+The price request contains the voucher code.
+
+If enabled in the [configuration](#configuration), an additional line with negative quantity is also sent.
+The ERP must respond with negative cost.
+
+The ERP can send a message that the voucher is invalid. In that case this message is displayed in the basket.
```
Comment (dabrt) [inline docs/guide/vouchers/vouchers.md:12]:
```suggestion
The ERP can send a message that the voucher is invalid. 
When this happens, a message is displayed in the basket.
```

File: docs/guide/vouchers/vouchers.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+## Configuration
+
+``` yaml
+twig:
+    globals:
+        voucher_active: true
+```
+
+Using the `send_vouchers_as_lines` setting you can configure whether the voucher is sent to ERP as additional line with negative quantity (`true`)
```
Comment (dabrt) [inline docs/guide/vouchers/vouchers.md:22]:
```suggestion
With the `send_vouchers_as_lines` setting, you can configure whether the voucher is sent to ERP as an additional line with negative quantity (`true`)
```

**Review by dabrt — CHANGES_REQUESTED** [review]

File: docs/guide/payment/payment.md [hunk 6]
```diff
@@ -1,41 +1,27 @@
...
-``` yaml
-jms_payment_core:
-    encryption:
-        provider: defuse_php_encryption
-        secret: 'def0000033444be8556a1bd7e347a240d19d34a078112f3f18bbb74cb4caeff9f9df7e2f1c86f32826b6b262360791264a0aeb851bdb999f2b882038448966b3b1d40a79'
-        enabled: true
-```
+Individual payment options / payment methods are implemented in Symfony bundles.
+As soon as they are registered in the kernel, they are displayed in the checkout payment form.
+The respective options are registered in the checkout form by a DIC compiler pass and must fulfill some requirements:
```
Comment (DominikaK) [inline docs/guide/payment/payment.md:21]:
```suggestion
The respective options are registered in the checkout form and must fulfill some requirements:
```

[the remark this answers is not in the document]
Comment (DominikaK) [reply docs/guide/payment/payment.md:21]: Good point

**Review by dabrt — APPROVED** [review]

**Pull request merged**